integration/container/ps_test.go

```diff
@@ -0,0 +1,79 @@
+package container
+
+import (
+	"context"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPsFilter(t *testing.T) {
+	defer setupTest(t)()
+	client := request.NewAPIClient(t)
+	ctx := context.Background()
+
+	prev, err := client.ContainerCreate(ctx,
+		&container.Config{
+			Cmd:   []string{"top"},
+			Image: "busybox",
+		},
+		&container.HostConfig{},
+		&network.NetworkingConfig{},
+		"prev",
+	)
+	require.NoError(t, err)
```

Comment:
Maybe move this into a helper so that the test can read:

```go
prev := createContainerForFilter(t, "prev")
top  := createContainerForFilter(t, "top")
next  := createContainerForFilter(t, "next")
```

Reply:
@dnephin Updated. Thanks.